python/Ganga/new_tests/TestProxy.py

```diff
@@ -69,9 +70,14 @@ def test_dict_attributes(self):
 
         # Proxy class instance
         self.assertFalse(hasattr(self.p, '_proxyClass'))
-        self.assertFalse(hasattr(self.p, '_proxyObject'))
+        self.assertTrue(hasattr(self.p, '_proxyObject'))
         self.assertTrue(hasattr(self.p, '_impl'))
 
+        # Proxied class
+        self.assertTrue(hasattr(self.p_class, '_proxyClass'))
+        self.assertFalse(hasattr(self.p_class, '_proxyObject'))
+        self.assertFalse(hasattr(self.p_class, '_impl'))
+
```

Comment:
I'm happy to remove this if the object is being parsed into an instance but I thought we were running these tests as unittests without the full machinery of Ganga which pulls in string parsing from the interpretor?
I'll remove the test save confusion here and write another test to see what's going on with objects when they're constructed.